test/unit_test/cli/test_cli.py

```diff
@@ -251,5 +251,49 @@ def test_cloud_cache_command(_, mock_container_client, test_set):
     mock_container_client().delete_blob.assert_called_once()
 
 
+@pytest.mark.parametrize("algorithm_names", [{"awq"}, {"awq", "gptq"}])
+@patch("olive.workflows.run")
+@patch("olive.cli.finetune.tempfile.TemporaryDirectory")
+@patch("huggingface_hub.repo_exists")
+def test_quantize_command(mock_repo_exists, mock_tempdir, mock_run, algorithm_names, tmp_path):
+    # some directories
+    tmpdir = tmp_path / "tmpdir"
+    tmpdir.mkdir()
+
+    output_dir = tmp_path / "output_dir"
+
+    # setup
+    mock_repo_exists.return_value = True
+    mock_tempdir.return_value = tmpdir.resolve()
+    mock_run.return_value = {}
+
+    for algo_name in algorithm_names:
```

Comment:
related to the above comment. this nesting doesn't exist if there is only one pass_flow https://github.com/microsoft/Olive/blob/180dffb8af7643d4dfcffcb044a14ba171ab2647/olive/engine/engine.py#L250

Reply:
The generated config never has more than one pass_flow and the output directories are created as expected. It is verified as part of the unit test as well.

Reply:
Nevermind, I think you mean in the output folder of the cli, not in the output folder of the olive_run.